docs/io/visualization/generating_widgets.ipynb

```diff
@@ -5,16 +5,17 @@
    "metadata": {},
```

Comment:
<p>These links don't work for me. Might be a simple rebasing issue</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
@MarkMageeAstro This is the same link which is present in the latest `master` branch as well.
https://tardis-sn.github.io/tardis/using/visualization/using_widgets.html

Reply:
<p>Okay, can you open an issue?</p><p> </p>

Reply:
I've solved the issue in this PR: https://github.com/tardis-sn/tardis/pull/2533